server/src/main/java/org/elasticsearch/search/aggregations/AggregationBuilders.java

```diff
@@ -266,7 +266,7 @@ public static GeoTileGridAggregationBuilder geotileGrid(String name) {
      * Create a new {@link SignificantTerms} aggregation with the given name.
      */
     public static SignificantTermsAggregationBuilder significantTerms(String name) {
-        return new SignificantTermsAggregationBuilder(name, null);
+        return new SignificantTermsAggregationBuilder(name);
```

Comment:
I'm slightly surprised to not have found a method in this collection for creating a `RareTermsAggregationBuilder`, but there isn't one.

Reply:
Undoubtedly just because I forgot :)  I always forget this helper class, since we only use it for tests ourselves and I prefer to construct objects directly instead of with the helper... which leads me to forget it :/